mfr/core/utils.py

```diff
@@ -63,6 +63,51 @@ def make_exporter(name, source_file_path, output_file_path, format):
             }
         )
 
+def bind_render(metadata, file_stream, url, assets_url, export_url):
```

Comment:
Note: need to remove the old make_renderer/make_exporter utils in favor of the ones that treat export and render as functions.